app/domain/authentication/authn_jwt/validate_status.rb

```diff
@@ -76,15 +77,10 @@ def validate_webservice_is_whitelisted
       end
 
       def validate_secrets
-        validate_signing_key_secrets
         validate_issuer
         validate_identity_secrets
       end
```

Comment:
aybe split `validate_secrets` to `validate_issuer` and `validate_identity_secrets` now that it doesnt validate all of the secrets